airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql

```diff
@@ -20,6 +20,10 @@
     cross join unnest({{ array_col }}) as {{ array_col }}
 {%- endmacro %}
 
+{% macro oracle__cross_join_unnest(stream_name, array_col) -%}
+
```

Comment:
Since we're not supporting unnest with oracle at first, can you put something like

```suggestion
    {% do exceptions.warn("Normalization does not support unnesting for Oracle yet.") %}
```